src/vr/vrObjectControls.js

```diff
@@ -204,7 +204,7 @@ export class vrObjectControls {
       v1 = v1.normalize();
 
       var mio = new Matrix4();
-      mio.getInverse(obj3d.matrixWorld);
+      mio.copy(obj3d.matrixWorld).invert();
```

Comment:
is this a file you wrote or is it copied from some library?

Reply:
wrote

Reply:
vr support in this lib is pretty much dead at the moment anyway